### ISSUE TITLE
Support for command options

### ISSUE THREAD

**Issue by abdulwahid** [stepanenko3/nova-command-runner#2]
Hi,

Thank you for this awesome package.

I just wanted to ask if you are planning to add support for command `options`  as well via `variables` soon.
The `variables` work perfectly when used for command `arguments` but not for `options`.

We have a scenario in which we are using command `options` to take start date and end date.
Sample command:

```
command:test-command {--startDate=default} {--endDate=default}
```

Now in config, if we simply add following:

```
'Test Command' => [
            'run' => 'command:test-command {--startDate=default} {--endDate=default}',
            'type' => 'primary',
            'group' => 'Test'
        ],
```

It takes input value for `--startDate` and `--endDate`, but we also need to append prefix `--startDate=` and `--endDate=` with date values to make it work.

![image](https://user-images.githubusercontent.com/4362643/164074305-d9f0418b-b726-4d8e-9f75-3247e8fd9c68.png)

Also we can't use `field => "date"` in this scenario, which is usable if we would have been using `arguments`.

**Comment by stepanenko3** [comment]
@abdulwahid

We support options to configure them in your example, use the following code
This will create a `log:demo --startDate=2022-04-19 --endDate=2022-04-21` command

```php
'Demo Log' => [
    'run' => 'log:demo --startDate={startDate} --endDate={endDate}',
    'type' => 'primary',
    'group' => 'Text',
    'variables' => [
        [
            'label' =>  'startDate',
            'field' => 'date',
        ],
        [
            'label' =>  'endDate',
            'field' => 'date',
        ]
    ],
],
```

**Comment by abdulwahid** [comment]
@stepanenko3 
Thanks for your quick response.
But this combination doesn't seem to be possible in Laravel command signature.
Every option or argument must be enclosed in curly braces, we can't define `--startDate=` without curly braces in command signature.
The example you provided throws error:

```
The "--startDate" option does not exist. 
```

because it tries to find `{--startDate}` in command signature which we don't have, we have it without braces `--startDate`.

**Comment by abdulwahid** [comment]
@stepanenko3 Can I create a PR to for this support?

**Comment by stepanenko3** [comment]
@abdulwahid,

Yes, it will be great if you open a PR

**Comment by abdulwahid** [comment]
@stepanenko3 

I am unable to push my branch because of permissions.


```
ERROR: Permission to stepanenko3/nova-command-runner.git denied to abdulwahid.
fatal: Could not read from remote repository.
```

**Comment by abdulwahid** [comment]
In `src/Dto/CommandDto.php`:

I changed from:

```
$command->setParsedCommand(
                            str_replace('{'.$variable['label'].'}', $variable['value'], $command->getParsedCommand() )
                        );
```

to: 

```
if (strpos($variable['label'], '--') !== false) {
                        $variableName = explode('=', $variable['label'])[0];
                        $command->setParsedCommand(
                            str_replace('{'.$variable['label'].'}',  $variableName . '=' .$variable['value'], $command->getParsedCommand() )
                        );

                    } else {
                        $command->setParsedCommand(
                            str_replace('{' . $variable['label'] . '}', $variable['value'], $command->getParsedCommand())
                        );
                    }
```

**Comment by nathan-io** [comment]
Hi @stepanenko3 ! Any plans to add this or adjust the repo permissions so a PR can be made?

**Comment by stepanenko3** [comment]
@nathan-io, @abdulwahid I released new version [4.0.11](https://github.com/stepanenko3/nova-command-runner/releases/tag/v4.0.11), which supports this
